libraries/emfl_platform_api/api.class.inc

```diff
@@ -148,9 +156,14 @@ class Emfl_Platform_API {
     } elseif( function_exists('curl_init') ) {
       // Fall back on CURL
       $curl = curl_init($url);
+      $authorization = 'Bearer ' . $this->access_token;
+
       curl_setopt($curl, CURLOPT_HEADER, false);
       curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
-      curl_setopt($curl, CURLOPT_HTTPHEADER, array("Content-type: application/json"));
+      curl_setopt($curl, CURLOPT_HTTPHEADER, array(
+      	  "Content-type: application/json",
+	      "Authorization: $authorization"
```

Comment:
@emfluenceindia does this header need the "Bearer" in it that the other headers have?

Reply:
Ahh never mind, I ignored the change on line 159.